CreeDictionary/API/search/types.py

```diff
@@ -1,80 +1,24 @@
-from typing import NamedTuple, Union, Set, NewType, Iterable, Tuple, cast
+from __future__ import annotations
 
-import attr
-from attr import attrs
+import dataclasses
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Union, NewType, Iterable, Tuple, Optional
 
 from typing_extensions import Protocol
 
-from API.helpers import serialize_definitions
-from API.models import Wordform, Definition
-from API.schema import SerializedLinguisticTag, SerializedSearchResult
-from utils import Language
+from API.models import Wordform
+from API.schema import SerializedLinguisticTag
 from utils.fst_analysis_parser import LABELS
-from utils.types import ConcatAnalysis, FSTTag
+from utils.types import FSTTag
 
 # it's a str when the preverb does not exist in the database
```

Comment:
Is this comment still accurate/relevant?

Reply:
Good catch, thank you!